src/init.cpp

```diff
@@ -1256,10 +1256,12 @@ bool AppInitMain(NodeContext& node)
     InitSignatureCache();
     InitScriptExecutionCache();
 
-    LogPrintf("Using %u threads for script verification\n", nScriptCheckThreads);
     if (nScriptCheckThreads) {
+        LogPrintf("Using %u threads concurrently for script verification\n", nScriptCheckThreads);
         for (int i=0; i<nScriptCheckThreads-1; i++)
             threadGroup.create_thread([i]() { return ThreadScriptCheck(i); });
+    } else {
+        LogPrintf("Using the only thread for script verification\n");
```

Comment:
Something feels a bit off with the wording here. What about "Using one thread for script verification"?

Reply:
It's not just any thread though, it's the main thread. No additional threads are spawned. I think that's what he's trying to convey.

Reply:
"Validation concurrency disabled: using the main thread for script validation."

Reply:
Isn't this the thread:
https://github.com/bitcoin/bitcoin/blob/feb1a8c03aff0d37730d80eee7a05cc7e4729850/src/init.cpp#L1267
?

Reply:
That spawns the scheduler thread, which is unrelated to script verification.

Reply:
On `-par=1`, it will inline the calls into the current thread. See also:

```
git grep '\-par=' test